source/common.c

```diff
@@ -40,7 +40,10 @@ struct aws_allocator *aws_default_allocator() {
 }
 
 void *aws_mem_acquire(struct aws_allocator *allocator, size_t size) {
-    return allocator->mem_acquire(allocator, size);
+    void * mem = allocator->mem_acquire(allocator, size);
+    if (!mem)
```

Comment:
braces or same line

Reply:
If we're going to make this change, it needs to be done EVERYWHERE we allocate memory

Reply:
I thought I did handle this everywhere we allocate memory. If I missed something, please point me to it.

Reply:
you're right I thought we also had it in the priority queue and scheduler implementations.